static/src/javascripts/projects/commercial/modules/cmp/lotame-cmp.js

```diff
@@ -0,0 +1,102 @@
+// @flow
+import { local } from 'lib/storage';
+
+type LotameError = {
+    error: number,
+};
+
+type LotameConsentType = {
+    name: string,
+    consent: boolean,
+};
+
+type LotameClientConsent = {
+    clientId: string,
+    lastupdate: number,
+    types: Array<LotameConsentType>,
+};
+
+type LotameSuccess = {
+    consent: Array<LotameClientConsent>,
+};
+
+const clientId: number = 12666;
+const lotameVendorId: number = 95;
+const lotameConsentKey: string = 'lotameConsent';
+
+const lotameConsentData = (isConsenting: boolean) => ({
+    analytics: isConsenting,
+    crossdevice: isConsenting,
+    datasharing: isConsenting,
+    targeting: isConsenting,
+});
+
+const getLotameConsent = (): number => local.get(lotameConsentKey);
```

Comment:
This only appears to be used as a boolean flag so could it return a boolean instead?

Reply:
Yeah, it could just be a boolean. I just kept it as a number as it is smaller.